src/network/types.rs

```diff
@@ -225,3 +225,36 @@ pub struct TeardownPortForward {
     // remove network related information
     pub complete_teardown: bool,
 }
+
+//  SetupNetwork contains options for setting up a container
+#[derive(Clone, Debug, Serialize, Deserialize)]
```

Comment:
remove Serialize Deserialize from the structs, we do not want to marshal/unmarshal them